src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -850,10 +868,14 @@ TieBreakMode getTiebreakMode() {
     return mode == null ? TieBreakMode.MODE_UNKNOWN : mode;
   }
 
-  Integer getRandomSeed() {
+  private String getRandomSeedRaw() {
     return rawConfig.rules.randomSeed;
   }
 
+  Integer getRandomSeed() {
+    return Integer.parseInt(rawConfig.rules.randomSeed);
```

Comment:
why don't your `getX` methods reference the corresponding `getXRaw` methods?

Reply:
Good call, switching.